### PR TITLE
Calculate no-extra-semicolon position correctly, fixes #1382

### LINKED ISSUE #1382: no-extra-semicolons line number is incorrect
With stylelint 6.5.1 and stylelint-config-standard 8.0.0, the `no-extra-semicolons` rule appears to return the wrong line number for errors.

The following styles:
```
div a {
    float: left;
}

div {
    float: left;;
}
```
triggers an error at line 2, column 15, when the error is at line 6 column 15 (or maybe 14?).

### PR DESCRIPTION
**Pull request by epmatsw**

### CODE REVIEW

**Comment by davidtheclark** [comment]
@epmatsw Nice job tracking this down. Could you please add a note to the changelog describing the fix? Then we can merge.

**Comment by epmatsw** [comment]
No problem, went ahead and updated it. Let me know if you need anything else!

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 - Added: `at-rule-name-newline-after` rule.
 - Fixed: `function-url-quotes` ignores values containing `$sass` and `@less` variables.
 - Fixed: selector-targeting rules ignore Less mixins and extends.
+- Fixed: no-extra-semicolons reports errors on the correct line
```
Comment (jeddy3) [inline CHANGELOG.md]: @epmatsw Thanks for fixing this bug!

Sorry to be pedantic, but can we backtick the rule name and add a full stop please? Our release notes are generated from the changelog and so I don't want to forget to correct this before we do a release.

**Comment by davidtheclark** [comment]
I went ahead and fixed the changelog formatting.

**Pull request merged**